app/controllers/devise/registrations_controller.rb

```diff
@@ -112,7 +112,7 @@ def sign_up(resource_name, resource)
   # The path used after sign up. You need to overwrite this method
   # in your own RegistrationsController.
   def after_sign_up_path_for(resource)
-    after_sign_in_path_for(resource)
+    after_sign_in_path_for(resource) if is_navigational_format?
```

Comment:
Could we cover this change with tests?

Reply:
I looked up all the current methods in Devise that use 'is_navigational_format?' checks and I couldn't seem to find anything in those tests that specifically incorporate a test for that navigational check. In my previous pull requests for Devise, I wrote my tests based off similar structured methods and their corresponding tests. However, since I cannot find how the other methods using 'is_navigational_format?' are testing it,  I am not sure how to
incorporate additional tests for the method I added it to. Based on the issue conversation, i was under the impression that the method I changed used to have a 'is_navigational_format?' check and that it somehow got deleted by accident from a git reset.  So i simply
added it back to the method.

Reply:
Yep, we don't have tests for any of these so I think it's ok for now.